packages/button/button.scss

```diff
@@ -4,8 +4,11 @@
 
 @import "~@fremtind/jkl-core/_functions.scss";
 
-$button-height--large: 3.5rem;
-$button-height--small: 2.5rem;
+$button-height--large: rem(56px);
+$button-height--small: rem(48px);
+$button-height--xs: rem(40px);
```

Comment:
Hva med `$button-height--compact`? `$button-height--normal`, da åpner vi for en `$button-height--small-screen` i Fremtinden™